db/schema.rb

```diff
@@ -55,6 +55,46 @@
     t.index ["creator_id"], name: "index_documents_on_creator_id"
   end
 
+  create_table "event_document_approveds", force: :cascade do |t|
+    t.bigint "document_id"
+    t.bigint "user_id"
+    t.datetime "created_at", null: false
+    t.datetime "updated_at", null: false
+    t.index ["document_id"], name: "index_event_document_approveds_on_document_id"
+    t.index ["user_id"], name: "index_event_document_approveds_on_user_id"
+  end
+
+  create_table "event_document_publisheds", force: :cascade do |t|
+    t.bigint "document_id"
+    t.bigint "user_id"
+    t.string "review_state"
```

Comment:
I'm wondering if we could just store the whole document, so we don't have to worry about which attributes were 'in-play' at the time.

Reply:
`review_state` is a confusing name here, it's supposed to be something that indicates whether or not this document was force-published. I think we might make that a separate event, or a boolean on the data record.